sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowToStorageApiProto.java

```diff
@@ -370,10 +376,20 @@ static Object singularFieldToProtoValue(
       case "TIMESTAMP":
         if (value instanceof String) {
           try {
-            return ChronoUnit.MICROS.between(Instant.EPOCH, Instant.parse((String) value));
-          } catch (DateTimeParseException e) {
-            return ChronoUnit.MICROS.between(
+            // "12345667"
+            ChronoUnit.MICROS.between(
```

Comment:
Seems like the formatted datetime would be the more-common case, right? If so, then we should probably try to parse using the DateTimeFormatter first.